cobalt/build/configs/android-arm/args.gn

```diff
@@ -1,18 +1,3 @@
-import("//cobalt/build/configs/proprietary_codecs.gn")
-
-target_os = "android"
-target_cpu = "arm"
-is_cobalt = true
-
-# Overriding flag from //media/media_options.gni.
-# Cobalt doesn't use //third_party/ffmpeg.
-media_use_ffmpeg = false
-
-# TODO(b/378934120): Remove flags below this line
-
-# These flags come from build/config/clang/clang.gni
-clang_use_chrome_plugins = false
-enable_check_raw_ptr_fields = false
-
-# This flag comes from build/config/compiler/compiler.gni
-treat_warnings_as_errors = false
+import("//cobalt/build/configs/android_common.gn")
```

Comment:
It doesn't make much of a difference, but you could consider having android_common.gn import common.gn